### PR TITLE
Update data model

### PR DESCRIPTION
**Pull request by babastienne**

### CODE REVIEW

**Review by Chatewgne — APPROVED** [review]

**Review by Chatewgne — APPROVED** [review]

**Comment by submarcos** [comment]
split data model update from fixture fix. add test to ensure fixtures are ok

File: Makefile [hunk 1]
```diff
@@ -47,18 +47,18 @@ css:
 
 %.pdf:
 	mkdir -p docs/data-model
-	postgresql_autodoc -h localhost -u geotrek -d geotrekdb -t dot -m "$*_.*" --password=geotrek -s "public"
+	postgresql_autodoc -l /usr/share/postgresql-autodoc -h localhost -u geotrek -d geotrekdb -t dot -m "$*_.*" --password=geotrek -s "public"
```
Comment (submarcos) [inline Makefile:50]:
```suggestion
	postgresql_autodoc -l /usr/share/postgresql-autodoc -h localhost -u geotrek -d geotrekdb -t dot -m "$*_.*" --password=geotrek -s "public"
```
Why using ``-l /usr/share/postgresql-autodoc`` ?

Comment (submarcos) [reply Makefile:50]: no need to specify on my instance as it's default shared data for postgresql-autodoc

Comment (babastienne) [reply Makefile:50]: Ok, on ubuntu 22.04 ?

I installed autodoc especially for this purpose and by default it seems that the program was trying to reach another folder (as mentioned in this issue https://github.com/cbbrowne/autodoc/issues/26).
If this is only a bug on my computer I guess we should remove this commit then.

**Pull request merged**